### PR TITLE
testcase/task_manager : add new testcase for empty user data

### PR DESCRIPTION
**Pull request by hasw7569**
testcase/task_manager : add new testcase for empty user data

Add new testcase for empty user data

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a91dc2a91423a3eb5d329bb5fba58e66e37cae76] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a91dc2a91423a3eb5d329bb5fba58e66e37cae76] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@hasw7569 Let's write `testcase/task_manager: add new testcases for xx(), yy(), and etc`

**Comment by seinfra** [comment]
Target : [191bac45343dc7b46b55f5c902f813b9f05c46d8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [191bac45343dc7b46b55f5c902f813b9f05c46d8] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c [hunk 1]
```diff
@@ -1041,6 +1056,7 @@ void tm_utc_main(void)
 
 	utc_taskmanager_dealloc_broadcast_msg_n();
 	utc_taskmanager_dealloc_broadcast_msg_p();
+
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c]: Please remove unnecessary this empty line.

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c [hunk 2]
```diff
@@ -479,6 +479,20 @@ static void utc_taskmanager_unicast_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+static void utc_taskmanager_broadcast_empty(void)
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c]: It looks a part of `utc_taskmanager_broadcast_n`.

**Comment by seinfra** [comment]
Target : [cf2d7d20259d8b2b236a9643ff5db7d8539b73ef] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [cf2d7d20259d8b2b236a9643ff5db7d8539b73ef] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c2f69c04227e14edf2c6f38264a6ce1648b10aec] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c2f69c04227e14edf2c6f38264a6ce1648b10aec] - Code Rule Check (C++) OK.

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c (hunk 1 above)
Comment (hasw7569) [reply apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c]: modify

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c (hunk 2 above)
Comment (hasw7569) [reply apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c]: I agree that opinion.
I Move this code into utc_taskmanager_broadcast_n.

**Pull request closed without merging**